src/rnp/fficli.cpp

```diff
@@ -394,6 +395,8 @@ ffi_pass_callback_stdin(rnp_ffi_t        ffi,
 
     if (!strcmp(pgp_context, "protect")) {
         protect = true;
+    } else if (!strcmp(pgp_context, "add subkey")) {
+        add_subkey = true;
```

Comment:
This and following lines for `add_subkey` case should be covered with tests, too major functionality to skip.

Reply:
see `test_additional_subkeys_reuse_password` in cli_tests.py

Reply:
Thanks!